spec/features/user/user_can_connect_to_github_spec.rb

```diff
@@ -0,0 +1,24 @@
+require "rails_helper"
+
+context "as a logged in user" do
+  describe "when it connects to Github", :vcr do
+    it "can see its repository" do
+      user = create(:user)
+
+      visit '/'
+      click_on "Sign In"
+
+      fill_in 'session[email]', with: user.email
+      fill_in 'session[password]', with: user.password
+
+      click_on 'Log In'
+
+      visit "/dashboard"
+      expect(page).to_not have_content("battleship")
+      mock_auth_hash
```

Comment:
Create test to show that github information is not seen until the the oauth is used to sign in with github which then makes the information available.